docs/README-selectors.md

```diff
@@ -21,7 +21,7 @@ substitute the value selected in the selector. For example:
     x = f("<&executor-type>")
 
 When rendered, the `<&executor-type>` string will be replaced by the
-lower case version of the selected value in the immediately preceeding
+lower case version of the selected value in the immediately prceeding
```

Comment:
Should this be "preceding"?

Reply:
Never trust a human - it was the one I had to fix manually. Sorry for the typo :-)
```suggestion
lower case version of the selected value in the immediately preceding
```

Reply:
Not sure I trust a computer much more :)